integrations/extensions/starter-kits/google-search/README.md

```diff
@@ -0,0 +1,126 @@
+# Google custom search
+
+## Background
+
+This is a starter kit for accessing the Google "Custom Search" JSON API. The "Custom Search" API allows search over a website, collection of websites or the world wide web using the [Google Programmable Search Engine](https://developers.google.com/custom-search/docs/overview) which is a configurable search that allows you to customize search features based on your use-case.
+
+The OpenAPI spec in this starter kit includes the following endpoint:
+
+- `GET /customsearch/v1`: Search for content relevant to a given query over the entire web.
+- `GET /customsearch/v1/siterestrict`: Search for content relevant to a given query over a specific collection of websites.
+
+The endpoints are described in detail at:
+ 
+1. **Custom Search**: https://developers.google.com/custom-search/v1/reference/rest/v1/cse/list 
+2. **Custom Search Site Restricted**: https://developers.google.com/custom-search/v1/reference/rest/v1/cse.siterestrict/list
+
+The Site Restricted API is similar to the Custom Search JSON API except that it has no daily query [limit](https://developers.google.com/custom-search/v1/overview#pricing).
```

Comment:
Also the "Custom Search Site Restricted" API is only available for a programmable search that is restricted to only searching specific sites (10 or fewer).  In contrast, "Custom Search" can do full web search as I understand it.  This is mentioned in the same link but the wording is confusing.